src/libraries/Microsoft.XmlSerializer.Generator/src/GenerateNupkgProps.targets

```
@@ -19,6 +19,12 @@
   &lt;ItemGroup&gt;
     &lt;DotNetCliToolReference Include="$(PackageId)" Version="$(PackageVersion)" /&gt;
   &lt;/ItemGroup&gt;
+  &lt;PropertyGroup&gt;
+    &lt;SgenPackSerializer Condition="'%24(SgenPackSerializer)'==''"&gt;False&lt;/SgenPackSerializer&gt;
+  &lt;/PropertyGroup&gt;
+  &lt;ItemGroup Condition="%24(SgenPackSerializer)==True"&gt;
```

Should use single quotes around both sides of the equals comparison.
```suggestion
  &lt;ItemGroup Condition="'%24(SgenPackSerializer)'=='True'"&gt;
```

---

updated